patches/build-config-android-internal_rules.gni.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/build/config/android/internal_rules.gni b/build/config/android/internal_rules.gni
-index b849838c5642430959890e554821b4eea7916a7a..cafae6ee551f2a57e5ed9442ccf5dd27679bebb7 100644
+index b849838c5642430959890e554821b4eea7916a7a..4b7121c114256709d833a99b82c9e687af442037 100644
 --- a/build/config/android/internal_rules.gni
 +++ b/build/config/android/internal_rules.gni
-@@ -4190,6 +4190,7 @@ if (enable_java_templates) {
+@@ -3959,6 +3959,7 @@ if (enable_java_templates) {
+           "android/java/src/org/chromium/chrome/browser/tabmodel/AsyncTabParamsManagerImpl.kt",
+           "java/androidx/core/os/BuildCompat.kt",
+           "webengine_shell_apk/src/org/chromium/webengine/shell/*.kt",
++          "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/*.kt",
```

Comment:
`_kt_allowlist += brave_kt_allowlist`

Reply:
Fixed in https://github.com/brave/brave-core/commit/69016d4bdbaf790da85437d1dc36e70109e22f11